po/cs.po

```diff
@@ -0,0 +1,2770 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
Thanks, merged!  Please feel free to set yourself as the copyright owner for the translation file itself.  cheers!